<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="serverRadioButton">
        <property name="text">
         <string>I am a server</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="clientRadioButton">
        <property name="text">
         <string>I am a client</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>420</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>420</width>
        <height>170</height>
       </size>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLineEdit" name="ipLineEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>131</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>192.168.188.123</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>191</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Opponent's IP:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="portSpinBox">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>10</y>
          <width>71</width>
          <height>30</height>
         </rect>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>29992</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="synchroWordLineEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>211</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>171</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Synchronisation word:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>171</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Escape word:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="escapeWordLineEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>90</y>
          <width>211</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>161</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Scroll event:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="eventComboBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>211</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startPushButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopPushButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
